src/agent/Makefile

```diff
@@ -23,16 +23,26 @@ COMMIT_MSG = $(if $(COMMIT),$(COMMIT),unknown)
 # Exported to allow cargo to see it
 export VERSION_COMMIT := $(if $(COMMIT),$(VERSION)-$(COMMIT),$(VERSION))
 
+include ../../utils.mk
+
 EXTRA_RUSTFEATURES :=
 
 ##VAR SECCOMP=yes|no define if agent enables seccomp feature
 SECCOMP := yes
 
+##VAR WASM=yes|no define if agent enables wasm feature
+WASM := yes
```

Comment:
How much overhead does it bring in if enabled by default? I would wonder if we should disable it by default since wasm is a non-standard use case of kata containers. We can always switch to enabling it by default containers when wasm becomes more popular.

Reply:
The memory footprint is increased from 119405.75(http://jenkins.katacontainers.io/job/kata-containers-2.0-metrics-ubuntu-20-04-PR/3140/console) to 126325.1(http://jenkins.katacontainers.io/job/kata-containers-2.0-metrics-ubuntu-20-04-PR/3163/console).

What about kata supply a special wasm-enable image if wasm is set to disable by default?

Reply:
hmm, how about adding an `experimental` image that can include any non-production-ready agent features?